test/nonregression/testing_tools.py

```diff
@@ -35,10 +35,10 @@ def likeliness_measure(file1, file2, threshold1, threshold2, display=False):
     import matplotlib.pyplot as plt
 
     print(' ** comparing ' + os.path.basename(file1) + ' **')
-    data1 = nib.load(file1).get_data()
+    data1 = nib.load(file1).get_fdata()
```

Comment:
Todo for later: replace other `get_data()` by `get_fdata()` (cf [doc](https://nipy.org/nibabel/reference/nibabel.dataobj_images.html))

Reply:
Hi @alexandreroutier ,
I don't understand why you pointed this line. The change in line 38 already done!

Reply:
Sorry for the misunderstanding.

The `get_data()` methods are also present in Clinica ( warnings are silenced when running Clinica) so we will have to take care of them one day :)